tensorflow/lite/micro/kernels/PORTING.md

```diff
@@ -0,0 +1,251 @@
+# Porting Reference Ops from Lite to Micro
+
+This is a guide to porting reference ops from Lite to Micro. It explains,
+step-by-step, the recommended code changes and the process for submitting them
+for review and acceptance.  The process results in multiple pull requests, or
+PRs. Multiple, [small PRs][] are easier for the project to review and merge.
+
+[small PRs]: https://google.github.io/eng-practices/review/developer/small-cls.html
```

Comment:
I didn't know we could do this in markdown, its very cool!

Can I suggest moving all of these links to the top of the file? And add a comment that you are defining links that will be reused throughout the file (for folks like me who don't know about this feature)? If I am reading the markdown itself from top to bottom, its easier if I know what all links have been defined.

Reply:
Done.

I've never thought to put them at the top, but it looks fine. From what I see, people tend to think of reference-links as footnotes, and put them at the bottom. I sorta footnote-ed after each paragraph for proximity's sake, but I agree that turned out ugly.

Reply:
> I didn't know we could do this in markdown, its very cool!

+1 @advaitjain 🙂 cc @lamberta @MarkDaoust @anirudh161